Packs/ServiceDeskPlus/Integrations/ServiceDeskPlus/ServiceDeskPlus.yml

```diff
@@ -0,0 +1,860 @@
+category: Case Management
+commonfields:
+  id: ServiceDeskPlus
+  version: -1
+configuration:
+- display: 'Data Center Location: Select the domain location that is applicable for
+    you application'
+  name: server_url
+  options:
+  - United States
+  - Europe
+  - India
+  - China
+  - Australia
+  required: true
+  type: 15
+- additionalinfo: test additional
+  display: Client ID
+  name: client_id
+  required: true
+  type: 4
+- display: Client Secret
+  name: client_secret
+  required: true
+  type: 4
+- display: Refresh Token
+  name: refresh_token
+  required: false
```

Comment:
not required? what is the flow that this will be used?

Reply:
No, it isn't required. The user can enter only the client id and secret and then use the integration to create the refresh token that will be used in the instance. I will explain in more details when we talk.